# Disable automagic, otherwise wlroots will enable features if dependencies are present.
diff --git a/meson.build b/meson.build
index 6a94dee6..9608717d 100644
--- a/meson.build
+++ b/meson.build
@@ -39,7 +39,7 @@ add_project_arguments(
   ],
   language: 'cpp')
 
-wlroots = subproject('wlroots', default_options: ['examples=false'])
+wlroots = subproject('wlroots', default_options: ['backends=drm,libinput,x11','examples=false','renderers=gles2','xcb-errors=disabled'])
 have_xwlr = wlroots.get_variable('features').get('xwayland')
 xcb_dep = dependency('xcb', required: get_option('xwayland'))
 
